pkg/controller/csi_handler.go

```diff
@@ -93,6 +101,99 @@ func (h *csiHandler) Init(vaQueue workqueue.RateLimitingInterface, pvQueue workq
 	h.pvQueue = pvQueue
 }
 
+// ReconcileVA lists volumes from the CSI Driver and reconciles the attachment
+// status with the corresponding VolumeAttachment object. If the attachment
+// status of the volume is different from the state on the VolumeAttachment the
+// VolumeAttachment object is patched to the correct state.
+func (h *csiHandler) ReconcileVA() error {
+	ctx, cancel := context.WithTimeout(context.Background(), h.timeout)
```

Comment:
I see the `Run` func takes a stop channel rather than a context. That is worth fixing before or at the same time as plumbing context into this function.

Reply:
@misterikkit could you create an issue on this repo for that?

Reply:
https://github.com/kubernetes-csi/external-attacher/issues/198